qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -177,10 +178,28 @@ def __json_decode__(cls, value: Dict[str, Any]) -> "BasisGateLibrary":
 
 
 class FixedFrequencyTransmon(BasisGateLibrary):
-    """A library of gates for fixed-frequency superconducting qubit architectures.
+    r"""A library of gates for fixed-frequency superconducting qubit architectures.
 
     Note that for now this library supports single-qubit gates and will be extended
     in the future.
+
+    Provided gates:
+        - x: :math:`\pi` pulse around the x-axis.
+        - sx: :math:`\pi/2` pulse around the x-axis.
+        - y: :math:`\pi` pulse around the y-axis.
+        - sy: :math:`\pi/2` pulse around the y-axis.
+
+    Pulse parameters:
```

Comment:
Hmm, it is good to document the default values. The subtlety that I wanted to document with my previous comment was about how `amp` gets used when the user provides it as a default value. That the user provides the `x` default `amp` and the `sx` default `amp` gets set to half that value without linking.

Reply:
Done here: [32ab306](https://github.com/Qiskit/qiskit-experiments/pull/889/commits/32ab30631e901210a7b38cc51e1497b5eff6154c)